samples/cpp/benchmark_app/README.md

```diff
@@ -125,11 +125,12 @@ Options:
                                 estimations the number of streams should be set to 1.
     -nthreads "<integer>"       Optional. Number of threads to use for inference on the CPU (including HETERO and MULTI cases).
     -enforcebf16="<true/false>" Optional. By default floating point operations execution in bfloat16 precision are enforced if supported by platform.
-    -pin "YES"/"HYBRID_AWARE"/"NUMA"/"NO"
+    -pin ("YES"|"CORE")/"HYBRID_AWARE"/"NUMA"/("NO"|"NONE")
                                 Optional. Explicit inference threads binding options (leave empty to let the OpenVINO to make a choice):
                                 enabling threads->cores pinning ("YES", which is already default for a conventional CPU),
                                 letting the runtime to decide on the threads->different core types ("HYBRID_AWARE", which is default on the hybrid CPUs)
                                 threads->(NUMA)nodes ("NUMA") or completely disable ("NO") CPU inference threads pinning.
+    -infer_precision "i8"/"i16"/"f32"/"bf16" Optional. Hint to specifies inference precision
```

Comment:
I suppose it should has another format: `device_name:infer_precision1,device_name:infer_precision2`. Otherwise how it should work for MULTI, AUTO scenarions?

Reply:
BTW it is also applicable for py benchmark_app

Reply:
done

Reply:
This PR covers only c++ benchmark_app py should be covered in separate PR